docs/api/app.md

```diff
@@ -715,14 +715,21 @@ To set the locale, you'll want to use a command line switch at app startup, whic
 **Note:** When distributing your packaged app, you have to also ship the
 `locales` folder.
 
-**Note:** On Windows, you have to call it after the `ready` events gets emitted.
+**Note:** On Windows, you have to call it after the `ready` event is emitted.
 
 ### `app.getLocaleCountryCode()`
 
 Returns `string` - User operating system's locale two-letter [ISO 3166](https://www.iso.org/iso-3166-country-codes.html) country code. The value is taken from native OS APIs.
 
 **Note:** When unable to detect locale country code, it returns empty string.
 
+### `app.getSystemLocale()`
+
+Returns `string` - The current system locale, fetched using Chromium's `l10n_util` library.
+Possible return values are documented [here](https://source.chromium.org/chromium/chromium/src/+/main:ui/base/l10n/l10n_util.cc).
+
+**Note:** This API must be called after the `ready` event is emitted.
```

Comment:
Can you update the `app.getLocale` note to match this one?

Reply:
The `app.getLocale` API is older and doesn't have a check to ensure the caller is calling it after the `ready` event. The downside of adding the check is it might break platforms where one can get the correct result even before the `ready` event, but the upside is it could prevent [a few bug reports](https://github.com/electron/electron/issues?q=is%3Aissue+getLocale+is%3Aclosed+ready). If it is worth adding the check, I think that should go into a separate PR.

Reply:
I would still suggest updating the docs to state that the API must be called after the `ready` event, even if that isn't strictly enforced on macOS and Linux currently. I agree, though, that adding an enforcement check would be better to do in another PR.